### PR TITLE
Release 7

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: app/views/vocab_sheets/_vocab_sheet_item.html.haml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-%li.vocab_sheet_item.vocab_sheet_page_item.shadow.rounded{:style => "width:#{vocab_sheet_item_width}px;height:#{vocab_sheet_item_height}"}
-  = hidden_field_tag "item[#{vocab_sheet_item.id}][id]", vocab_sheet_item.id, :class => 'item_id'
-  = link_to sign_url(vocab_sheet_item.sign_id), {:class => 'drawing vocab_sheet_drawing', :style => "line-height:#{vocab_sheet_image_height}px;height:#{vocab_sheet_image_height}px"} do
```
Comment (elspeth-rabid) [inline app/views/vocab_sheets/_vocab_sheet_item.html.haml]: These inline styles with interpolation a pretty leftfield.
Definitely room to work on simplifying views and styles.
Will add to my Friday radar!

**Review by elspeth-rabid — COMMENTED** [review]
I agree with Hound. Good progress on improving the code base!!!

File: app/views/vocab_sheets/_vocab_sheet_item.html.haml (hunk 1 above)
Comment (Br3nda) [reply app/views/vocab_sheets/_vocab_sheet_item.html.haml]: Sweet... I like that the code base is a million times better than when we picked it up years ago..
the Haml needs some clean up


the JS is the worst though.

**Comment by Br3nda** [comment]
This needs some testing on staging.

**Review by elspeth-rabid — APPROVED** [review]

**Pull request merged**